File: JY/10월 3주차/n^2배열자르기.py
def solution(n, left, right):
    answer = []
    if left == right:
        return [max((left%n) , right//n)]
    
    for i in range(left, right+1):
        y_max =  (i%n) + 1
        x_max = (i//n) + 1
        # print(f"Y : {y_max}, X : {x_max}")
        answer.append(max(y_max, x_max))
    
    return answer

File: MK/10월 3주차/n^2배열자르기.java
class Solution {
    public int[] solution(int n, long left, long right) {

        int idx=0;
        int limit = (int) (right - left + 1);
        int[] ans = new int[limit];

        while (left <= right) {
            ans[idx++]=(int) Math.max(left/n,left%n)+1;
            left++;

        }

        return ans;
    }
}